index.js

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env node
+const _ = require('lodash');
 const colors = require('colors'); // eslint-disable-line no-unused-vars
 // this ~crazy-scary~ _amazing_ package mutates Strings globally
 const copyPaster = require('copy-paste');
 
-const supposedDate = process.argv[2];
-let date = new Date(supposedDate);
-if (!date.getDate()) date = new Date();
+const target = process.argv[2];
+const date = target ? new Date(target) : target;
```

Comment:
this breaks the no-input use case.

calling `todo-month` from the command line with no arguments should return your current month.